internal/sandbox/sandbox.go

```diff
@@ -85,11 +85,11 @@ type Sandbox interface {
 	// The container used to execute the command is reused until Clean() is
 	// called.
 	//
-	// If there is an error while using the sandbox an error will be returned.
+	// If an error occurs while using the sandbox, it will be returned.
 	//
 	// The result of the supplied command will be returned in an instance of
 	// RunResult.
-	Run(...string) (*RunResult, error)
+	Run(string, ...string) (*RunResult, error)
```

Comment:
I suggest naming these args to make the interface clearer:

e.g. `Run(command string, args ...string)`

Reply:
Will do, thanks